tests/en/light_HassTurnOn.yaml

```
@@ -31,26 +31,39 @@ tests:
         domain: light
     response: "Turned on the lights"
 
+    # Turn on all lights in the home
   - sentences:
-      - "turn on the lights"
-      - "turn on the lights in here"
-      - "turn the lights in here on"
-      - "turn the lights on here"
+      - Turn all lights on in the appartment
+      - Switch on every lights in the home
```

```suggestion
      - Switch on every light in the home
```

English speakers wouldn't use the plural "lights" in this or some of the other sentences.